crates/leon/src/values.rs

```diff
@@ -0,0 +1,37 @@
+use std::{
+    collections::{BTreeMap, HashMap},
+    hash::Hash,
+};
+
+pub trait Values<K, V> {
+    fn get_value(&mut self, _key: K) -> Option<V>;
```

Comment:
I think this is a typo:

```suggestion
    fn get_value(&mut self, key: K) -> Option<V>;
```

Reply:
It's not, but it doesn't really matter here anyway